### ISSUE TITLE
Cannot find jax checkpoint

### ISSUE THREAD

**Issue by PoodleWang** [salesforce/jaxformer#7]
Would you be able to provide the Jax checkpoint?

**Comment by enijkamp** [comment]
Yes, I still have to push the sharding patterns for the 16B forward-pass and then will push the weights. Sorry for the delays.

**Comment by glicerico** [comment]
any updates on sharing the 16B model  @enijkamp ?
